Marlin/src/lcd/menu/menu_main.cpp

```diff
@@ -182,7 +182,7 @@ void menu_main() {
       if (thermalManager.targetHotEnoughToExtrude(active_extruder))
         GCODES_ITEM(MSG_FILAMENTCHANGE, PSTR("M600 B0"));
       else
-        SUBMENU(MSG_FILAMENTCHANGE, [](){ _menu_temp_filament_op(PAUSE_MODE_LOAD_FILAMENT, 0); });
+        SUBMENU(MSG_FILAMENTCHANGE, [](){ _menu_temp_filament_op(PAUSE_MODE_PAUSE_PRINT, 0); });
```

Comment:
Although the old type was plain "pause" we now have an actual `PAUSE_MODE_CHANGE_FILAMENT`! That's what I'd meant to use here.